src/halmos/utils.py

```diff
@@ -313,23 +316,25 @@ def decode_hex(hexstring: str) -> Optional[bytes]:
         return None
 
 
-def hexify(x):
+def hexify(x, contract_name: str = None):
```

Comment:
By setting the default value of contract_name to None, existing usage of this function can use without modification